### PR TITLE
Upgrade json-cpp to 1.9.3.

### PR DESCRIPTION
**Pull request by ekpyron**
The release notes read generally favorable: https://github.com/open-source-parsers/jsoncpp/releases

And json-cpp 1.9.2 is slower to build and causing issues with clang-10 and the ossfuzz builds, stalling the docker image work of @aarlt.

### CODE REVIEW

**Review by axic — APPROVED** [review]
Looks okay to me, but perhaps wait for another review.

**Comment by ekpyron** [comment]
There shouldn't be any harm in doing this and I double checked that there's no mention of 1.9.2 left in the repo, so I think I'll go ahead merging this, once CI finishes, so that @aarlt can better proceed.

**Pull request merged**
